llmfoundry/utils/builders.py

```diff
@@ -701,7 +701,6 @@ def _validate_cfg(icl_cfg: dict[str, Any]):
     return evaluators, logger_keys
 
 
-@experimental_function('Tensor Parallelism')
```

Comment:
I think we should leave this personally. We may want to remove the experimental designation in Composer and Foundry at different times.

Reply:
+1

Reply:
added back